source/PreferencesPanel.cpp

```diff
@@ -508,6 +512,11 @@ void PreferencesPanel::DrawSettings()
 			isOn = true;
 			text = to_string(static_cast<int>(100. * Preferences::ViewZoom()));
 		}
+		else if(setting == SCREEN_MODE_SETTING)
+		{
+			text = Preferences::ScreenModeSetting();
+			isOn = true;
```

Comment:
```suggestion
			isOn = true;
			text = Preferences::ScreenModeSetting();
```
I'm being picky here, but to respect the convention we put isOn first if the preference does not matter for it.